vnext/PropertySheets/External/Microsoft.ReactNative.Cpp.Dependencies.props

```diff
@@ -11,7 +11,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup Label="CppWinRT">
-    <CppWinRTVersion>2.0.211028.7</CppWinRTVersion>
+    <CppWinRTVersion Condition="'$(CppWinRTVersion)' == ''">2.0.211028.7</CppWinRTVersion>
```

Comment:
We should probably validate a minimum version.
Versions earlier than this default have a packaging problem which made `PackageReference` fail.